module/importer/src/Domain/Command/Import/Attribute/ImportUpdateAttributesInProductCommand.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Importer\Domain\Command\Import\Attribute;
+
+use Ergonode\Core\Domain\ValueObject\TranslatableString;
+use Ergonode\Importer\Domain\Command\ImporterCommandInterface;
+use Ergonode\Product\Domain\ValueObject\Sku;
+use Ergonode\SharedKernel\Domain\Aggregate\ImportId;
+
+class ImportUpdateAttributesInProductCommand implements ImporterCommandInterface
```

Comment:
IMHO seeing the names of the other importing commands, name should be without Update eg.
ImportProductAttributesCommand , ImportProductAttributesValueCommmand or ImportAttributesValueCommand